README.md

```diff
@@ -1,3 +1,4 @@
+*For  English click [here: ](https://github.com/micverb/postman_voorbeelden/blob/main/README_ENG.md)*
```

Comment:
Zou je in plaats van deze regel het volgende bovenaan de readme kunnen zetten? Zo is de link relatief en valt hij meer op. Verder helemaal perfect!

# postman
## English version
*For  English click [__here__](README_ENG.md)*

## Introductie
Voorbeelden voor het gebruik van de FHIR API, gedocumenteerd als een Postman collectie. Importeer het JSON bestand in deze map in postman, en creëer een *environment* met variabelen *username* en *password*.